modin/pandas/base.py

```diff
@@ -70,7 +70,7 @@
 
 # Do not lookup certain attributes in columns or index, as they're used for some
 # special purposes, like serving remote context
-_ATTRS_NO_LOOKUP = {"____id_pack__", "__name__"}
+_ATTRS_NO_LOOKUP = {"____id_pack__", "__name__", "_cache"}
```

Comment:
Otherwise, it triggers index calculation, which is bad for lazy calculations (previously tests for hdk failed).